client/app/components/parameters.html

```diff
@@ -18,6 +18,7 @@
     >
       <i class="zmdi zmdi-settings"></i>
     </button>
-    <parameter-value-input param="param" on-change="onValuesChange" apply-button="true"></parameter-value-input>
+    <parameter-value-input param="param" />
```

Comment:
I saw some [inputs](https://github.com/angular/angular.js/issues/1953#issuecomment-13135021) against self-closing tags in angular, but we do have them in other parts of the code and no problems reported so far :man_shrugging:

Reply:
Oh ok it's actually not an Angular thing but a browser thing. I got used to it with jsx... I'll revert.

Reply:
Fixed